src/pylogctx/django.py

```diff
@@ -10,13 +10,13 @@
 logger = logging.getLogger(__name__)
 
 
-class ExtractRequestContextMiddleware(object):
+class OuterMiddleware(object):
     def process_request(self, request):
-        extractor = settings.PYLOGCTX_REQUEST_EXTRACTOR
+        context.clear()
         try:
-            context.update(**extractor(request))
-        except Exception:
-            logger.exception()
+            context.update(request)
+        except Exception as e:
```

Comment:
I think it is better not to catch all exceptions here because it's very easy to misconfigure your application and find it out after a week or two in production.
From the other hand it's very weird if your log_adapter raise an Error and this error will crush your app.
So may be we could something like this?
```
try:
    context.update(request)
except SomeLoggerAdapterNoSuchAdapterError:
    raise  # this is disputable, but this allows you to see configuration error immediately
except Exception as e:
    logger.exception("Failed to push request object to log context")  # because this is error and I want to see it as soon as possible (otherwise I will lose tons of logs)
```